source/_components/sensor.linky.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Linky"
+description: "Instructions on how to integrate Linky daily data within Home Assistant."
+date: 2018-09-06 08:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_release: 0.78
+ha_category: Energy
+ha_iot_class: "Cloud Polling"
+---
+
+
+The `linky` sensor platform is showing the last day comsuption of the house from the linky electric meter.
```

Comment:
`comsuption` -> `consumption`
`the linky electric` -> `the Linky electric`
`the house` -> `your home`